File: packages/insomnia-httpsnippet/test/fixtures/output/python/python3/cookies.py
import http.client

conn = http.client.HTTPConnection("mockbin.com")

headers = { 'cookie': "foo=bar; bar=baz" }

conn.request("POST", "/har", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: packages/insomnia-httpsnippet/test/fixtures/output/javascript/fetch/cookies.js
const fetchOptions = {
  mode: 'cors',
  method: 'POST',
  headers: {
    cookie: 'foo=bar; bar=baz',
  },
};

fetch('http://mockbin.com/har', fetchOptions)
  .then(response => response.json())
  .then(data => console.log(data));
